macros/src/attr/enum.rs

```diff
@@ -1,16 +1,17 @@
 use std::collections::HashMap;
 
+use proc_macro2::TokenStream;
 use syn::{Attribute, Ident, Result, Type, WherePredicate};
 
+use super::{parse_assign_from_str, parse_bound};
 use crate::{
     attr::{parse_assign_inflection, parse_assign_str, parse_concrete, Inflection},
     utils::{parse_attrs, parse_docs},
 };
 
-use super::parse_bound;
-
 #[derive(Default)]
 pub struct EnumAttr {
+    pub crate_rename: Option<TokenStream>,
```

Comment:
This should be an `Option<syn::Path>` to allow better parsing and compiler errors when invalid syntax is provided.